newdle/client/src/components/MyNewdles.js

```diff
@@ -100,11 +101,19 @@ function MyNewdle({newdle: {code, title, participants, duration, final_dt: final
         <Icon name="users" />
         <label>
           <em>
-            {!participants.length
-              ? 'There are no participants yet'
-              : answers.length !== participants.length
-              ? `${answers.length} out of ${participants.length} participants answered`
-              : 'All participants answered'}
+            {!participants.length ? (
+              t`There are no participants yet.`
+            ) : answers.length !== participants.length ? (
+              <Trans>
+                <Plural
+                  value={participants.length}
+                  one={`${answers.length} out of # participant answered`}
```

Comment:
Actually this case will never happen because you never get in this branch in case of `answers.length === participants.length`. Not sure if omitting the `one` prop works, but if yes we can probably do that

Reply:
If I remove it then newdles with 1 participant that don't have any answer yet get no text at all.

Reply:
Indeed, I thought we had a "Nobody has answered yet" case but we don't